### PR TITLE
Revamps Seed Vault Lavaland Ruin

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

This pull request changes the seed vault to look more like a vault, changing the design to make it look more visually appealing and easier to play on.

It also adds a few extra seed spawners and restocking parts for the vendors, water tanks, soap, and a loom!

Here's a look at the new and improved vault!
image

## Why It's Good For The Game

Should make the seed vault more enjoyable to play on and more interesting to explore. 

Gives the pod people more seeds to play around with, extending the playtime of the vault.

## Changelog
:cl:
add: Loom, soap, restocking parts, water tanks to the seed vault
add: An entirely re-designed seed vault
del: Removed the old seed vault
tweak: gave the seed vault more seeds to play around with

/:cl:

### CODE REVIEW

**Comment by CAPTTLasky** [comment]
This is actually my second PR
image

**Comment by 81Denton** [comment]
:^)

**Comment by Tharcoonvagh** [comment]
What a lad, I merely changed two words in a description and he remade fucking brig
Good job!

**Comment by kriskog** [comment]
Nice

File: code/modules/research/designs/autolathe_designs.dm [hunk 1]
```diff
@@ -279,6 +279,7 @@
 	materials = list(/datum/material/iron = 1000)
 	build_path = /obj/item/tank/internals/generic
 	category = list("initial","Misc","Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_CARGO
```
Comment (ATH1909) [inline code/modules/research/designs/autolathe_designs.dm]: uh, was this change supposed to be a part of this PR?

Comment (CAPTTLasky) [reply code/modules/research/designs/autolathe_designs.dm]: Is it a part of another merged PR because that's the only reason it should be?


AAAAAAAAAAAA

**Comment by AnonymousNow** [comment]
Looks kind of cool, but that one-tile-wide workspace around the most trafficked part of the vault is going to get right on the tits of the lifebringers if there's more than one of them.

File: code/modules/research/designs/autolathe_designs.dm (hunk 1 above)
Comment (CAPTTLasky) [reply code/modules/research/designs/autolathe_designs.dm]: Fixed

**Comment by CAPTTLasky** [comment]
I think I fixed it

**Comment by CAPTTLasky** [comment]
> 
> 
> Looks kind of cool, but that one-tile-wide workspace around the most trafficked part of the vault is going to get right on the tits of the lifebringers if there's more than one of them.

I kind of agree with you. I can expand the area inside to make it easier to work in.

**Comment by AnonymousNow** [comment]
I made a quick and dirty image edit, as a mockup of what I had in mind. Tell me what you think of it.
aaaa

**Comment by CAPTTLasky** [comment]
> 
> 
> I made a quick and dirty image edit, as a mockup of what I had in mind. Tell me what you think of it.
> aaaa

Check the updated main post

**Comment by AnonymousNow** [comment]
Ah, looks good. With a couple of pinch points instead of a corridor, it works much better.

**Comment by HomocidlePancakes** [comment]
Looks nice, but amma gonna nitpick. Trash can in being in the center of the room is silly. Move to a corner. Will make the room easier to modify too.

**Comment by CAPTTLasky** [comment]
> 
> 
> Looks nice, but amma gonna nitpick. Trash can in being in the center of the room is silly. Move to a corner. Will make the room easier to modify too.

It's different and it doesn't look bad, plus isn't really a functional issue. Modifying a simple disposal chute is pretty simple and I'm not worried about it. Throwing it in a corner might make it less conspicuous but I want it to be conspicuous.

**Comment by HomocidlePancakes** [comment]
Fairnuff.

**Comment by TheVekter** [comment]
> This is actually my second PR

image

**Comment by trollbreeder** [comment]
Well done CAPTTLasky, especially for an up and coming contributor!

**Comment by IndieanaJones** [comment]
You should give them Kudzu

**Comment by CAPTTLasky** [comment]
If a maintainer is ok with me giving them kudzu I will but otherwise this is ready.

**Comment by kriskog** [comment]
They can make kudzu, no reason to give it to them. Ill take a look soonish.

**Pull request merged**